plugins/modules/apt_rpm.py

```diff
@@ -28,6 +28,8 @@
   package:
     description:
       - List of packages to install, upgrade, or remove.
+      - May include paths to local C(.rpm) files if C(state=installed|present),
```

Comment:
```suggestion
      - Since community.general 8.0.0, may include paths to local C(.rpm) files if O(state=installed) or O(state=present),
```
(line might now be too long :) )